### PR TITLE
Release notes placeholder PR

### PR DESCRIPTION
**Pull request by disa6302**
*Issue #, if available:*

*Description of changes:*

Placeholder PR. Only the README.md will be merged on approval

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,12 @@ export LD_LIBRARY_PATH=/<YOUR_PRODUCER_SDK_CPP_DOWNLOAD>/amazon-kinesis-video-st
 This should resolve native library loading issues.
 
 ## Release Notes
+
+### Release 2.0.0 (27 May 2020)
+* Migration to new CPP Producer SDK base
+* Updated docker scripts to incorporate the changes in producer SDK
+* Provision of CachedInfoMultiAuthServiceCallbacks callback to use different credential providers for different streams in one client with existing stream information
```
Comment (sharadrj) [inline README.md:139]: @disa6302 , can you point me to the code/PR which added Multi Auth Callback? I would like to understand it better.

Comment (disa6302) [reply README.md:139]: @sharadrj , 

This is the one: 
https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-java/pull/88

File: README.md [hunk 2]
```diff
@@ -132,6 +132,11 @@ export LD_LIBRARY_PATH=/<YOUR_PRODUCER_SDK_CPP_DOWNLOAD>/amazon-kinesis-video-st
 This should resolve native library loading issues.
 
 ## Release Notes
+
+### Release 2.0.0 (27 May 2020)
```
Comment (MushMal) [inline README.md:136]: Is there a good reason to bump up the major version? Perhaps 1.10.0?

File: README.md [hunk 3]
```diff
@@ -132,6 +132,11 @@ export LD_LIBRARY_PATH=/<YOUR_PRODUCER_SDK_CPP_DOWNLOAD>/amazon-kinesis-video-st
...
+* Updated docker scripts to incorporate the changes in producer SDK
+* Provision of CachedInfoMultiAuthServiceCallbacks callback to use different credential providers for different streams in one client with existing stream information
```
Comment (MushMal) [inline README.md:138]: Could you perhaps add a little more context like:

Provision of 'CachedInfoMultiAuthServiceCallbacks' callbacks which implements caching layer and per-stream auth which is useful in scenarios where a single client object is used for multiple streams that are intended for different accounts. This is useful for service proxy type of scenarios.


Please reword this as you see fit

File: README.md (hunk 2 above)
Comment (disa6302) [reply README.md:136]: The reason for the major revision bump up is because the producer SDK is bumped up. Bumping up to establish that our base SDK has gone through a major rev. But, yes. Other than the docker scripts breaking between revs, I do not see anything else that might break. Let me change it to 1.10.0.

File: README.md (hunk 3 above)
Comment (disa6302) [reply README.md]: Sure!

File: README.md (hunk 2 above)
Comment (MushMal) [reply README.md]: I am somewhat split and gravitate more towards not changing the major version. The minor version update is for the C producer and CPP producer as they do take advantage of the functionality that we added in lower layers and some are not fully compatible. Whereas Java SDK is based on PIC and should be fully compatible.
